app/views/pages/layouts/default.liquid.haml

```diff
@@ -18,12 +18,14 @@ is_layout: false
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/button.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/card.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/dropdown.min.css' | stylesheet_tag }}
+    {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/form.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/icon.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/input.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/label.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/menu.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/popup.min.css' | stylesheet_tag }}
     {{ '//oss.maxcdn.com/semantic-ui/2.1.8/components/transition.min.css' | stylesheet_tag }}
+    {{ '//oss.maxcdn.com/semantic-ui/2.1.8/semantic.min.css' | stylesheet_tag }}
```

Comment:
@tap87 Did you mean to add the full collection of semantic-ui styles? This is overriding much of our site specific styles, most noticeably `font-size` across the site.